client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-details.tsx

```diff
@@ -1,21 +1,20 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { getPlan, PLAN_BUSINESS, PLAN_BUSINESS_MONTHLY } from '@automattic/calypso-products';
 import { CloudLogo, Button, PlanPrice } from '@automattic/components';
+import { SiteDetails } from '@automattic/data-stores';
+import { useSitePlans } from '@automattic/data-stores/src/plans';
```

Comment:
I haven't tested it fully yet, but as an early comment, I think it's not entirely ideal to consume something from the `src` directory of a package. These are usually not intended to be distributed and can often cause build and other issues.

WDYT about doing it in an alternative way like in the following image?-
<img width="1411" alt="Screenshot 2024-06-26 at 9 04 12 AM" src="https://github.com/Automattic/wp-calypso/assets/6820724/8e337bfa-423d-4694-b212-881e5debdf2b">

Reply:
Thanks @Imran92! I'll apply your suggestion.

To justify my import a bit, here is where it is originated from: https://github.com/search?q=repo:Automattic/wp-calypso+@automattic/data-stores/src/&type=code

Reply:
Updated here: https://github.com/Automattic/wp-calypso/pull/92088/commits/0bab0b32c58c7086fd0b4d12b09d2c1a43cbe3ea